code/go/main-workshop/go.mod

```
@@ -0,0 +1,10 @@
+module cdk-workshop
+
+go 1.16
```

Maybe make this 1.18 already (1.16 and 1.17 are EOL).

```suggestion
go 1.18
```